v3/lints/cabf_smime_br/lint_subscribers_crl_distribution_points_are_http_test.go

```diff
@@ -0,0 +1,40 @@
+package cabf_smime_br
+
+import (
+	"testing"
+
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/test"
+)
+
+func TestSubscriberCrlDistributionPointsAreHTTP(t *testing.T) {
+	testCases := []struct {
+		Name           string
+		InputFilename  string
+		ExpectedResult lint.LintStatus
+	}{
+		{
+			Name:           "pass - cert with HTTP CRL distribution point",
+			InputFilename:  "smime/subscriber_with_http_crl_distribution_point.pem",
+			ExpectedResult: lint.Pass,
+		},
+		{
+			Name:           "error - cert without a non-HTTP CRL distribution point",
+			InputFilename:  "smime/subscriber_with_non_http_crl_distribution_point.pem",
+			ExpectedResult: lint.Error,
+		},
+		{
+			Name:           "error - cert without no HTTP CRL distribution points",
+			InputFilename:  "smime/legacy_subscriber_with_non_http_crl_distribution_point.pem",
+			ExpectedResult: lint.Error,
+		},
+	}
```

Comment:
Can you also add a test case for a legacy certificate with one http CRL distribution point and one non-HTTP distribution point?  It wouldn't hurt to add a failing case on non-legacy with the same setup although it's not _strictly_ required it would show that your lint is effectively covering it's _multipurpose_(s)

Reply:
I commented "Done" on your other comment twice when I meant to comment it on both comments (including this one). I'm still kinda new to GitHub.